pkg/api/controller_test.go

```diff
@@ -2408,7 +2489,7 @@ func TestAuthorizationWithOnlyDefaultPolicy(t *testing.T) {
 		So(resp.StatusCode(), ShouldEqual, http.StatusForbidden)
 
 		if entry, ok := conf.AccessControl.Repositories[TestRepo]; ok {
-			entry.DefaultPolicy = []string{"create", "read"}
+			entry.AnonymousPolicy = []string{"create", "read"}
 			conf.AccessControl.Repositories[TestRepo] = entry
 		}
 
```

Comment:
there are some SetBasicAuth(username, passphrase). below this line, remove them as they are not needed for this test.